### ISSUE TITLE
Need to be able to split out validation data for better training

### ISSUE THREAD

**Issue by matsen** [matsengrp/torchdms#39]
For example, we want to train several times and pick the fit that does the best in terms of validation loss.
